app/models/photo_comment.rb

```diff
@@ -0,0 +1,3 @@
+class PhotoComment < ApplicationRecord
+  belongs_to :photo
```

Comment:
@ddosque we couldn't figure out how to reconcile database conflicts - e.g. comments have a relationship with posts and photocomments have a relationship with photos...we thought it was cleaner to have them clearly separated